docs/contributors/testing.md

```diff
@@ -108,7 +108,7 @@ tests are supposed to fail in that case).
 <br/><br/>
 * You have added an interface for handling interactions that manipulates the visible diagram.
 <br/><br/>
-`bpmn.navigation.test.ts` drives all visual tests for that part. A visual test only requires a BPMN diagram as input.
+`diagram.navigation.zoom.pan.tests.ts` drives all visual tests for that part. A visual test only requires a BPMN diagram as input.
```

Comment:
I just saw there is a missing part to explain what is the `diagram.navigation.fit.tests.ts`, `generated.svg.test.ts` and `overlays.rendering.test.ts` tests 🙁

Reply:
I decided to add those in this PR so we don't waste time